plugins/admin/README.adoc

```diff
@@ -120,15 +120,15 @@ Use "kn admin autoscaling [command] --help" for more information about a comman
 ====
 ----
 $ kn admin domain set --custom-domain mydomain.com
-Updated Knative route domain mydomain.com
+Set Knative route domain mydomain.com
 ----
 ====
 
 .Update a custom domain with --selector and Service with a label app=v1 will use test.com
 ====
 ----
 $ kn admin domain set --custom-domain test.com --selector app=v1
-Updated Knative route domain test.com
+Set Knative route domain test.com with selector [app=v1]
```

Comment:
Same as above add “to”

Reply:
@maximilien 
As Knative supports multiple domains, I'm wondering if `set route domain to` is more like Knative only support one domain?

Reply:
IMO if use "set" it implies setting the single value. Otherwise, I would expect an "Add route".

How would one manage multiple routes here ? If this is only about a 'default' route maybe "Set default Knative route domain ..." is better choice.

Also, I wonder whether the "Knative" is not redundant and maybe we should talk about a "service" instead of a "route" here, as route is an internal concept mostly hidden in kn. We are always talking about a service's or  revsion's URL.

Reply:
@rhuss 
In fact, I think in this context it is more about `domain` other than `route`.

Knative support to have multiple domains as below, in kn-admin we want to help with add and remove domains. I'm thinking maybe `kn admin domain add` is more proper here and perhaps also change `route domain` to `domain` as it may be a little bit confusing with the Knative Route resource.

@rhuss @maximilien what do you think? Thanks!

```
data:
  # example.org will be used for routes having app=prod.
  example.org: |
    selector:
      app: prod
  # Default value for domain, for routes that does not have app=prod labels.
  # Although it will match all routes, it is the least-specific rule so it
  # will only be used if no other domain matches.
  example.com: ""
```

Reply:
Yeah, the "route" confused me a bit. Maybe just referring to the domain without any revision/service prefix makes more sense. 

For this example, "Add" makes more sense as it also emphasises that you add a domain with the selector.

For general wording of the command itself (set/unset), it's a bit tricky.
As you can have multiple things that can be identified, having "create/delete" is typically a better wording for commands than "set/unset" as this more implies a single entity to manage. This is how we do it for the Knative entities like service or sources.

However, as you also can override with "set" then "add" would be misleading. Not sure if an "create/update/delete" makes sense, as this would add a possibly too fine-grained workflow (and would require the user to know whether a domain already is adder).

I hope we can implement the knative story that introduces an "apply" verb soon, which would be useful here, too: "apply" essentially mean, create-or-update and is idempotent (you can call it with the same argument many times, but only get a single result). This is the case for you command here.

So, if this were a core kn command, I probably would suggest a 

```
# Add or update the given domain with the given selector
kn admin domain apply ....

# Remove the domain
kn admin domain remove ....
```

as this aligns with the other verbs used in kn.

Reply:
@rhuss I feel the same way. kn-admin itself should be consistent, now we have add/update/etc. This need refine and perhaps this change can wait for and follow `kn client` story you mentioned.
How about I open another chore item to follow it and we can complete this PR first.

Reply:
IMO, set/unset suit better for properties like domain, 
while create/update/apply/deploy operates on the actual resources.

Reply:
@navidshaikh thanks and we can continue subcommand wording in #53